app/wp-content/themes/kstg/library/scss/components/_footer.scss

```diff
@@ -1,8 +1,11 @@
-
-
 footer {
-  height: 59px;
+  .inner-footer {
+    max-width: 1220px;
+    margin: 0 auto;
+  }
+  height: 63px;
   bottom: 0;
+  margin-top: -16px;
```

Comment:
I may be missing something here, but I would avoid a negative margin if you can. One, it's just one of those things to avoid where possible. More importantly, you may have some issues on your other pages if your footer is creeping up into the section above it, forcing you to always have to put extra padding on what's above. 

I would just modify the padding on the `.aside-wrapper` class (probably in the homepage scss?)